src/frontend/src/flows/addDevice/manage/addDeviceSuccess.test.ts

```diff
@@ -0,0 +1,61 @@
+import { renderAddDeviceSuccess } from "./addDeviceSuccess";
+import copyJson from "./addDeviceSuccess.json";
+
+describe("addDeviceSuccess", () => {
+  const deviceAlias = "Test device alias";
+
+  const { en } = copyJson;
+
+  const waitFor = (callback: () => void): Promise<void> =>
+    new Promise((resolve) => {
+      const start = Date.now();
+
+      const callCallback = () => {
+        if (Date.now() > start + 10 * 1000) {
+          resolve();
+          return;
+        }
+
+        try {
+          callback();
+          resolve();
+        } catch (err: unknown) {
+          // Ignore error and try again in a bit
+          setTimeout(callCallback, 100);
+        }
+      };
+
+      callCallback();
+    });
+
+  beforeAll(() => {
+    const pageContent = document.createElement("div");
+    pageContent.setAttribute("id", "pageContent");
+    document.body.appendChild(pageContent);
+  });
+
+  it("should render static content", async () => {
+    renderAddDeviceSuccess({
+      deviceAlias,
+    });
+
+    await waitFor(() => expect(document.body.textContent).toContain(en.title));
+    expect(document.body.textContent).toContain(en.explore);
+  });
+
+  it("should render a call to action", () => {
+    renderAddDeviceSuccess({
+      deviceAlias,
+    });
+
+    expect(document.querySelector("button")).not.toBeNull();
+  });
```

Comment:
What does this test? Won't this break if we ever replace the button with a link? Is the test worth having?

Reply:
Not a must but it covers exactly what you describe. The one that modify the HTML element is made aware that it was an explicit choice to have use a button - i.e. we use a button because we want to programmatically navigate to avoid to add something in the navigation history. I can also add a test that particularly test that but, stopped before 😉.

Reply:
Removed